.circleci/config.yml

```diff
@@ -1,31 +1,53 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@0.1.0
+  plugin-ci: mattermost/plugin-ci@volatile
 
 workflows:
   version: 2
-  deploy:
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - master
     jobs:
       - plugin-ci/lint
       - plugin-ci/test
       - plugin-ci/build
-      - plugin-ci/deploy-release:
+  ci:
+    jobs:
+      - plugin-ci/lint:
           filters:
             tags:
               only: /^v.*/
+      - plugin-ci/coverage:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/build:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/deploy-ci:
```

Comment:
Does this mean `deploy-ci` will run on every branch and every PR?

Reply:
No, it only runs on `master` as configured in L36-37.